fastcrypto/Cargo.toml

```diff
@@ -35,10 +35,10 @@ generic-array = { version = "0.14.6", features = ["serde"] }
 typenum = "1.15.0"
 # TODO: switch to https://github.com/celo-org/celo-bls-snark-rs
 # when https://github.com/celo-org/celo-bls-snark-rs/issues/228 is solved
-celo-bls = { git = "https://github.com/huitseeker/celo-bls-snark-rs", branch = "updates-2", package = "bls-crypto" }
+celo-bls = { git = "https://github.com/huitseeker/celo-bls-snark-rs", branch = "updates-2-with-parallelism-toggle", package = "bls-crypto", default-features = false }
```

Comment:
Is there a way to point to a non personal branch?

Reply:
No. https://github.com/celo-org/celo-bls-snark-rs/issues/228